lib/manager/helmv3/extract.ts

```diff
@@ -0,0 +1,76 @@
+import is from '@sindresorhus/is';
+import yaml from 'js-yaml';
+import * as datasourceHelm from '../../datasource/helm';
+import { logger } from '../../logger';
+import { SkipReason } from '../../types';
+import { ExtractConfig, PackageDependency, PackageFile } from '../common';
+
+export function extractPackageFile(
+  content: string,
+  fileName: string,
+  config: ExtractConfig
+): PackageFile | null {
+  let chart;
+  try {
+    chart = yaml.safeLoad(content, { json: true });
+    if (!(chart?.apiVersion && chart.name && chart.version)) {
+      logger.debug(
+        { fileName },
+        'Failed to find required fields in Chart.yaml'
+      );
+      return null;
+    }
+    if (chart.apiVersion !== 'v2') {
+      logger.debug(
+        { fileName },
+        'Unsupported Chart apiVersion. Only v2 is supported.'
+      );
+      return null;
+    }
+  } catch (err) {
+    logger.debug({ fileName }, 'Failed to parse helm Chart.yaml');
+    return null;
+  }
+  let deps = [];
+  if (!(chart && is.array(chart.dependencies))) {
+    logger.debug({ fileName }, 'Chart has no dependencies');
+    return null;
+  }
+  deps = chart.dependencies.map((dep) => {
+    const res: PackageDependency = {
+      depName: dep.name,
+      currentValue: dep.version,
```

Comment:
Should we first filter chart.dependencies for entries that have both `depName` and `currentValue`?

Reply:
Maybe not wrong as an additional check. See https://github.com/renovatebot/renovate/pull/6820/files#diff-ad106bf0f585aa336023b11d5cc3073bR45